src/app/clusters/scenes-client/scenes-client.c

```diff
@@ -39,7 +39,7 @@
  ******************************************************************************/
 
 #include "scenes-client.h"
-#include "../../include/af.h"
+#include "af.h"
```

Comment:
I think we should be using the chip include path for these, since they're compiled by applications against the CHIP tree.

recommend
```
#include <app/util/af.h>
```